pypesto/hierarchical/inner_calculator_collector.py

```diff
@@ -248,7 +248,14 @@ def get_inner_par_ids(self) -> list[str]:
         ]
 
     def get_interpretable_inner_par_ids(self) -> list[str]:
-        """Return the ids of interpretable inner parameters of all inner problems."""
+        """Return the ids of interpretable inner parameters of all inner problems.
+
+        Interpretable parameters need to be easily interpretable by the user.
+        Examples are scaling factors, offsets, or noise parameters. An example
+        of non-interpretable inner parameters is the spline heights of spline
+        approximation for semiquantitative data. It is challenging to interpret
+        the meaning of these parameters based solely on their value.
+        """
```

Comment:
Yo could link to the documentation of the base problem, but not necessary. Would save some lines and guarantee a common description.